docs/admin/Metadata/index.rst

```diff
@@ -0,0 +1,115 @@
+Daarnaast bestaat er de mogelijkheid om voor de eigen organisatie *extra informatiecategorieën* toe te voegen. Deze zijn te herkennen aan de waarde "Zelf-toegevoegd item" in de kolom `Oorsprong` van de lijst. Deze kunnen indien nodig gewijzigd worden.
+
+Op het beheerscherm zijn de volgende acties mogelijk:
+
+* Rechtboven zit een knop **"informatiecategorie toevoegen"** waarmee een *informatiecategorie* toegevoegd kan worden.
+* Bovenaan zit een zoekveld met en knop **"Zoeken"** waarmee naar *informatiecategorieën* gezocht kan worden.
+* Daaronder zit de mogelijkheid om **eenzelfde actie uit te voeren over meerdere informatiecategorieën**. Op dit moment wordt alleen de actie **"Geselecteerde informatiecategorieën verwijderen"** ondersteund. Merk op dat het mogelijk is om in de lijst één of meerdere *informatiecategorieën* aan te vinken.
+* Onder de (bulk-)actie staat de lijst met *informatiecategorieën*. Door op de kolomtitels te klikken kan de lijst **alfabetisch of chronologisch geordend** worden. 
+* Rechts naast de lijst bestaat de mogelijkheid om deze te **filteren op oorsprong**.
+* Bij een *informatiecategorie* kan op de `naam` geklikt worden om **de details in te zien** en deze eventueel **te wijzigen**.
+* Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de `audittrail`_ in te zien.
+
+Wanneer bij een *informatiecategorie* op  de `naam` wordt geklikt, wordt een scherm geopend met nadere details.
+Hierop zien we:
+
+* **Alle gegevens**. Deze lichten we hieronder toe.
+* Rechtsboven een knop **"Show logs"**. Deze toont de volledige `audittrail`_ van de *informatiecategorie*.
+* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de Admin-interface zijn uitgevoerd op de *informatiecategorie*.
+* Als de *informatiecategorie* tot de landelijke 17 behoort, linksonder een knop om het scherm te **sluiten** en teurg te keren naar de lijst.
+* Als de *informatiecatgeorie* door de organisatie zelf is toegevoegd, linksonder de mogelijkheid om **wijzigingen op te slaan** (indien van toepassing). Er kan voor gekozen worden om na het opslaan direct een nieuwe *informatiecategorie* aan te maken of om direct de huidige *informatiecategorie* nogmaals te wijzigen.
+* Rechtsonder de mogelijkheid om de *informatiecategorie* te **verwijderen**.
+
+Op een informatiecategorie zijn de volgende gegevens beschikbaar. Op het scherm wordt verplichte velden **dikgedrukt** weergegeven.
+
+* ``Naam``. De naam van de *informatiecategorie*, bijvoorbeeld "convenant".
+* ``Naam meervoud``. De meervoudsvorm van de naam, bijvoorbeeld "convenanten".
+* ``Definitie``. De (mogelijk landelijke) definitie van de *informatiecategorie*.
+* ``UUID``. Een niet-muteerbaar, automatisch toegekend identificatie kenmerk.
+* ``Oorsprong``. Een niet-muteerbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
```

Comment:
```suggestion
Naam
    De naam van de *informatiecategorie*, bijvoorbeeld "convenant".

Naam meervoud
    De meervoudsvorm van de naam, bijvoorbeeld "convenanten".

Definitie
    De (mogelijk landelijke) definitie van de *informatiecategorie*.

UUID
    Een niet-wijzigbaar, automatisch toegekend identificatiekenmerk.

Oorsprong
    Een niet-wijzigbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
```

* ben niet zeker of deze deifnition list syntax per se beter is, laat ik aan jou om te bepalen
* muteerbaar -> wijzigbaar ivm. toegankelijk taalgebruik waarbij eenvoudigere woorden de voorkeur hebben.